packs/lastline/actions/lib/analysis_apiclient.py

```diff
@@ -0,0 +1,1890 @@
+#!/usr/bin/python
```

Comment:
Why do you need this file? It's an extra lib I can't `pip` install because crappy upstream.